src/affirmations.js

```diff
@@ -111,6 +111,36 @@ const affirmations = [
       "she is water\n soft enough\n to offer life\n tough enough\n to drown it away",
     source: "unknown",
     author: "rupi kaur"
+  },
+  {
+    affirmation:
+      "I am free of the fear of mistakes and failure. Mistakes and failures are good and necessary to achieve success. I act without fear of failure and feel excited, heroic, and unstoppable",
+      source: "unknown",
+      author: "Corey Regan"
+  },
+  {
+    affirmation:
+      "Wealth is constantly flowing into my life.",
+      source: "unknown",
+       author: "unknown"
+  },
+  {
+    affirmation:
+      "Cherish your visions and dreams as they are the children of your soul, the blueprints of your ultimate achievements.",
+      source: "unknown",
+      author: "unknown"
+  },
+  {
+    affirmation:
+      "Nothing is impossible. The word itself says " I'm possible!" "
```

Comment:
```suggestion
      "Nothing is impossible. The word itself says \"I'm possible!\" "
```

The quotes in this string need to be escaped so that JavaScript treats them like a string and not special characters. You can find more information about this here: https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/escaping-literal-quotes-in-strings/